src/api/libcellml/component.h

```diff
@@ -444,6 +450,8 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
 
     bool doIsResolved() const override; /**< Virtual method for implementing isResolved, @private. */
 
+    bool doEqual(const EntityPtr &other) const override; /**< Virtual implementation method for equals, @private. */
```

Comment:
This may be the case but I don't think it is the role of this PR to address this.  If indeed it is seen as an issue then one should be created.  I don't see this PR being responsible for addressing this issue.

Reply:
Agreed, I was merely "reminding" ourselves of the issue (which we have discussed before). I have therefore updated issue #402 to account for this particular issue.